### DISCUSSION TITLE
活塞破聯盟最高的連敗數

### DISCUSSION THREAD

**Discussion by Tomken123** [1101-GS4719-C/Python-Community-GS4719-C#1813] [General]
我是中央第一cade粉，網路上一堆人在臭他水貨狀元，但是他真的水嗎?先不說活塞除了bojan一個空間點都沒有，其他人的進攻手段也都很單一，除了cade以外也沒有其他強點，酸民都只看數據，看他失誤數很多，就認為他不太會控球，投籃命中率低，就覺得他外線都練不起來，可憐!若是他到聯盟頂尖強隊，也一定會是主力球星，也不用忍受你們這些酸民的嘴臉!!!!!!!

**Comment by uwje** [comment]
活塞真的爛，聚集眾多天賦出眾的球員，卻打出爛出天際的戰績，說個笑話，他們的勝場數，甚至少於剛剛復出的賈莫蘭特（因為持槍禁賽），用」所以希望活塞能找一個有經驗的球員來引導他們來打球，畢竟能在這裡打的球員技術都不錯，但心態差很多
